SU-SWS/saa_victoria_subtheme
# ADAPT-476: Added base input style fix.

# READY FOR REVIEW
# Summary
- adds back in base style settings being overridden by decanter/tailwind base styles.

# Review By (Date)
- 7/19

# Urgency
- low but noticeable

# Steps to Test

1. pull down branch
1. clear caches
2. check input in sign up form in footer to see that it has correct padding.
3. check any other inputs/forms/text areas - i don't know of any others on the site, do you cc @rebeccahongsf ?

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket 476 testing
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
